app/web/src/utils/names.ts

```diff
@@ -1,3 +1,7 @@
 export function firstName(fullName?: string) {
   return fullName?.split(" ")[0] ?? "";
 }
+
+export function acronym(name?: string) {
+  return name?.trim()?.slice(0, 1).toUpperCase();
+}
```

Comment:
We can remove this function if using the `Avatar` component we have already as mentioned in the `Navigation` file.